source/_integrations/remote.markdown

```diff
@@ -11,14 +11,18 @@ ha_codeowners:
 ha_integration_type: entity
 ---
 
-Keeps track of the remotes in your environment. Captures their state and allows you to control them.
+The **Remote** {% term integration%} keeps track of the remotes in your environment. Captures their state and allows you to control them.
```

Comment:
I'd make this one sentence, with comma, instead of period.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/34791